index.js

```diff
@@ -0,0 +1,90 @@
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project3: {
+    title: 'Project Name goes here',
+    img: './images/card-three.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project4: {
+    title: 'Project Name goes here',
+    img: './images/card-four.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project5: {
+    title: 'Project Name goes here',
+    img: './images/card-five.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project6: {
+    title: 'Project Name goes here',
+    img: './images/card-six.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+};
+
+const btns = document.querySelectorAll('.btn');
+btns.forEach((b) => {
+  b.addEventListener('click', () => {
+    const parent = b.parentNode;
+    const div = document.createElement('div');
+    div.className = 'popup';
```

Comment:
- [ ] As requested by the previous reviewer, I would kindly suggest you follow the Figma template as much as possible for your modal pop-up design

Below 👇  is a video explanation of what you need to work on

[screen-capture (3).webm](https://user-images.githubusercontent.com/78634841/211008759-03a755de-a4ef-4482-a23c-7341316df24b.webm)